recipe/bld.bat

```diff
@@ -1,18 +1,11 @@
-set LIB=%LIBRARY_LIB%;%LIB%
-set LIBPATH=%LIBRARY_LIB%;%LIBPATH%
-set INCLUDE=%LIBRARY_INC%;%INCLUDE%;%RECIPE_DIR%
 
-set GEOS_LIBRARY_PATH=%LIBRARY_BIN%\geos_c.dll
+set GEOS_LIBRARY_PATH=%LIBRARY_BIN%
 
-del /f shapely\speedups\_speedups.c
-del /f shapely\vectorized\_vectorized.c
```

Comment:
I would prefer these two lines to remain, so that the `.c` files are always cythonized by conda's version. For instance `Shapely-1.8.1.tar.gz` has inconsistent copies of these files:

- has `shapely/speedups/_speedups.c`, so not modified (i.e. not cythonized by conda's version)
- does not have `shapely/vectorized/_vectorized.c`, so we see "cythoning shapely/vectorized/_vectorized.pyx to shapely/vectorized\_vectorized.c" in the log

There was a bit of discussion at https://github.com/shapely/shapely/pull/1264

Reply:
Thanks for the feedback, my understanding was that these files were zapped upstream and no longer present in the source distribution artifacts (1.8.2+).  Whilst running in CI here, they weren't doing anything as the files were not present.  I'll add it back in as it is a harmless noop without them present anyway.